.github/workflows/reusable-fuzz.yml

```diff
@@ -83,14 +83,14 @@ jobs:
         id: fuzz
         shell: bash {0}
         env:
-          DURATION: ${{ inputs.duration }}
+          FUZZ_ITERATIONS: ${{ inputs.iterations }}
           FUZZ_SHELL: ${{ inputs.shell }}
           TARGET: ${{ matrix.target }}
         run: |
-          timeout "${DURATION}s" npm run fuzz "${TARGET}"
+          npm run fuzz "${TARGET}"
```

Comment:
This change _could_ be reverted since the old duration-based (using `timeout`) approach will still work with the new fast-check iteration-based approach.

Reply:
I'm going to revert this change since the iteration-based approach is too variable in terms of time, ranging from 5 minutes (no shell on Unix) to >60 minutes for `10_000` iterations (csh and PowerShell).